### DISCUSSION TITLE
KERI Implementer's Call 07-16-2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#91] [Meetings]
Reports
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann 
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

**Comment by pfeairheller** [comment]
## Signifypy

- No new PRs or Issues

**Comment by rodolfomiranda** [comment]
## KERIA
### PRs
- https://github.com/WebOfTrust/keria/pull/269
- https://github.com/WebOfTrust/keria/pull/268
- https://github.com/WebOfTrust/keria/pull/267

### No new issues

**Comment by m00sey** [comment]
KERIpy:

Vote to close:
- https://github.com/WebOfTrust/keripy/pull/793
- https://github.com/WebOfTrust/keripy/pull/796

Changes requested:
- https://github.com/WebOfTrust/keripy/pull/806
- https://github.com/WebOfTrust/keripy/pull/816

Closed:
- https://github.com/WebOfTrust/keripy/pull/815
- https://github.com/WebOfTrust/keripy/pull/817

**Comment by henkvancann** [comment]
We've split up old WOT-terms repo into:

- [kerisse](https://github.com/WebOfTrust/kerisse) : search engine (Typesense-based [github.io site](https://weboftrust.github.io/kerisse/))
- new [WOT-terms](https://github.com/WebOfTrust/WOT-terms): glossary, solely focussed on terminology, filtering, and connect to source management (Docusaurus-based [github.io site](https://weboftrust.github.io/WOT-terms/?level=2))
- [keridoc:](https://github.com/WebOfTrust/keridoc) KERI documentation site (Docusaurus-based: [github.io site](https://weboftrust.github.io/keridoc/?level=2))

Apart from this, we still have:
- [kerific](https://github.com/WebOfTrust/kerific): SSI-terminology dictionary including KERI terms (browser extension for Brave/Edge/Chrome).

**Reply by pfeairheller** [reply]
I made the updates

**Comment by lenkan** [comment]
# Signify-ts

No new PRs or issues. 

Some old PRs need merging:

- https://github.com/WebOfTrust/signify-ts/pull/267 Should unblock the rest when merged.

**Reply by lenkan** [reply]
Question about new recipient field: https://github.com/WebOfTrust/signify-ts/pull/267#discussion_r1642575432

**Reply by lenkan** [reply]
Also related: https://github.com/trustoverip/tswg-keri-specification/pull/183
